.github/workflows/staging-odfe-cli.yml

```diff
@@ -0,0 +1,64 @@
+name: Staging ODFE CLI
+
+on:
+  schedule:
+    - cron: '0 12 * * *'
+  repository_dispatch:
+    types: [staging-odfe-cli]
+
+jobs:
+  Test-ODFE-CLI:
+    runs-on: ubuntu-18.04
+    name: Test-ODFE-CLI
+    strategy:
+      matrix:
+        go-version: [1.14]
+    steps:
+      - name: Set up AWS Cred
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - uses: actions/checkout@v1
+      - name: Required Packages
+        run: ./release-tools/scripts/required_packages.sh
+      - name: Retrieve plugin tags
+        run: |
+          yq --version
+          echo "p_tag_cli=$(./release-tools/scripts/manifest_parser.sh opendistro-odfe-cli plugin_version)" >> $GITHUB_ENV
+      - name: Checkout odfe-cli
+        uses: actions/checkout@v2
+        with:
+          repository: opendistro-for-elasticsearch/odfe-cli
+#          ref: ${{env.p_tag_cli}}
+          ref: main
```

Comment:
Revert to `ref: ${{env.p_tag_cli}}` after testing is done

Reply:
The main ref will remain as the CLI team still doesn't have the 1.0.0 tag cut. I will uncomment the ref: ${{env.p_tag_cli}} once they move to match their artifacts with their repo tag.